src/main/java/org/isf/admission/gui/AdmissionBrowser.java

```diff
@@ -1873,8 +1782,7 @@ private JButton getSaveButton() {
 
 					// get delivery date
 					if (deliveryDateFieldCal.getDate() != null) {
-						deliveryDate = new GregorianCalendar();
-						deliveryDate.setTime(deliveryDateFieldCal.getDate());
+						deliveryDate = deliveryDateFieldCal.getLocalDateTime();
```

Comment:
It's very strange, checks fail because codebase seems not having this change, even if I checkout your PR locally (and I have to apply myself). I think the PR check is confused.

I try to merge it as is, eventually I fix later.